takwimu/dataset_release_utils.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings
```

Comment:
In order to maintain wazimap structure, lets move and rename this file to `data/utils.py`.